DESCRIPTION

```diff
@@ -17,8 +17,11 @@ Depends:
     R (>= 3.6),
     rClr (>= 0.9.1)
 Imports: 
+    dplyr,
+    purrr,

```

Comment:
@Yuri05 Can we use those packages?

Reply:
@IndrajeetPatil Not all packages are available for us as they need to be validated in the ValidR environment

Reply:
Yes, both packages are available in the ValidR.
Current ValidR-Release:
dplyr: 0.8.5
purrr: 0.3.3

Upcoming ValidR-Release (full packages list s. [here](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/wiki/ValidR-4.0.4-packages-list)):
dplyr: 1.0.5
purrr: 0.3.4